CHANGELOG.md

```diff
@@ -4,10 +4,12 @@ All notable changes to this project will be documented in this file.
 ## [Unreleased]
 
 ### Added
+- `background` keyword in `uvdata.set_lsts_from_time_array` to allow lst calculation in a background thread.
```

Comment:
Just a quick note: related to this I think we should also mention that we've added the `background_lsts` keyword for the data-reading methods and it's defaulted to `True`.